src/Symfony/Bridge/Monolog/Tests/Processor/TokenProcessorTest.php

```diff
@@ -48,7 +52,7 @@ public function testProcessor()
             $this->markTestSkipped('This test requires symfony/security-core 5.3+');
         }
 
-        $token = new UsernamePasswordToken('user', 'password', 'provider', ['ROLE_USER']);
+        $token = new UsernamePasswordToken(new InMemoryUser('user', 'password', ['ROLE_USER']), 'provider', ['ROLE_USER']);
```

Comment:
Why do we need `if (class_exists(InMemoryUser::class))` in `SwitchUserTokenProcessorTest` but not here?

Reply:
Because line 52 makes sure this test only runs on Symfony 5.3 (which has this new user classname)